git-subhistory.sh

```diff
@@ -351,7 +356,7 @@ cd ./$(git rev-parse --show-cdup) || exit $?
 GIT_DIR="$(git rev-parse --git-dir)"
 
 # a temporary directory for e.g. filter-branch backups
-mkdir "$GIT_DIR/subhistory-tmp/" || exit $?
+mkdir -p "$GIT_DIR/subhistory-tmp/" || exit $?
```

Comment:
although I have thought that some way to override this to use e.g. RAM disk instead would be nice. Doesn't need to be an option, obscure enough to simply be an env variable:

```sh
TMP_DIR=/Volumes/RAMdisk git-subhistory merge whatever whatever

test -z "$TMP_DIR" && TMP_DIR="$GIT_DIR/subhistory-tmp/" || {
  test -d "$TMP_DIR" || die '$TMP_DIR ('"$TMP_DIR"') is not a directory'
}
```